shared/chat/routes.tsx

```diff
@@ -1,51 +1,96 @@
 import {isMobile} from '../constants/platform'
 
+import ChatConversation from './conversation/container'
+import ChatEnterPaperkey from './conversation/rekey/enter-paper-key'
```

Comment:
Does doing this import here negate the dynamic loading of the Component? What does the compiled js look like for this?

Reply:
in my testing it did not. since we just pull it in and use it in typeof its only visible to TS